source/write-scripts.txt

```diff
@@ -434,6 +434,53 @@ The following lines are equivalent.
    space it will not be saved in your command history. This minimizes
    exposure if you enter passwords on the command line.
 
+Execute Code From a Configuration File
+--------------------------------------
+
+On startup, :binary:`mongosh` checks for a file named
+:ref:`.mongoshrc.js <mongoshrc-js>`. If :ref:`.mongoshrc.js
+<mongoshrc-js>` is found, :binary:`mongosh` interprets the file before
+displaying the prompt for the first time.
+
+.. seealso::
+
+   :ref:`.mongoshrc.js <mongoshrc-js>`, :ref:`mongosh-shell-settings`
+
+Execute JavaScript Code
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/examples/ex-display-line-numbers.rst
+
+Execute MongoDB Code
+~~~~~~~~~~~~~~~~~~~~
+
+To create a log of when your :binary:`mongosh` client connects to a
+database, add the following to ``<your-home-directory>/.mongoshrc.js``:
+
+.. code-block:: javascript
+
+   db.clientConnections.updateOne(
+      { "_id": ObjectId() },
+      { $currentDate: { "connectTime": true } },
+      { upsert: true }
+   ) ;
```

Comment:
Is there a reason that this isn’t `db.clientConnections.insertOne({ connectTime: ISODate() })`?

Reply:
Thanks for the suggestion, updated.